playground/background.ts

```diff
@@ -26,9 +27,13 @@ cameraEntity.addComponent(OrbitControl);
 
 engine.run();
 
+engine.canvas._webCanvas.addEventListener("onresize", () => {
+  engine.canvas.resizeByClientSize();
+});
+
 engine.resourceManager
   //@ts-ignore
-  .load<TextureCubeMap[]>([
+  .load<[TextureCubeMap, TextureCubeMap, Texture2D]>([
```

Comment:
Maybe this is a wrong grammar.

Reply:
It is not a wrong grammer. It is just a generic.